test/cmd/newapp.sh

```diff
@@ -50,6 +52,23 @@ os::cmd::expect_failure 'oc get dc/mysql'
 os::cmd::expect_failure 'oc get dc/php'
 os::cmd::expect_success_and_text 'oc new-app -f test/testdata/template-without-app-label.json -o yaml' 'app: ruby-helloworld-sample'
 
+# check object namespace handling
+# hardcoded values should be stripped
+os::cmd::expect_success_and_not_text 'oc new-app -f test/testdata/template-with-namespaces.json -o yaml' 'namespace: STRIPPED'
```

Comment:
Use of `-o jsonpath` would be a more precise way to verify the namespace field for all of these new additions.